FIP-0063: Tweak protocol change details (#928)
filecoin-project/FIPs: FIPS/fip-0063.md

### Block Headers

As today, blocks should include beacon entries for the epoch corresponding to their own height, as well as those corresponding to any epoochs that were null.

> This guarantees that a unique beacon entry can be found within the Filecoin blockchain to serve as a source of external randomness to the protocol.

Reviewer: ```suggestion
This guarantees that a unique beacon entry can be found for any Filecoin epoch to serve as a source of external randomness to the protocol.
```
